CommandLine.c

```diff
@@ -0,0 +1,371 @@
+/*
+htop - CommandLine.c
+(C) 2004-2011 Hisham H. Muhammad
+(C) 2020-2021 htop dev team
+Released under the GNU GPLv2, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "config.h" // IWYU pragma: keep
+
+#include <assert.h>
+#include <getopt.h>
+#include <locale.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+#include <unistd.h>
+
+#include "Action.h"
+#include "CRT.h"
+#include "CommandLine.h"
```

Comment:
The `#include "CommandLine.h"` should go right after `#include "config.h"`.

Reply:
> LGTM. Although the noise in `Makefile.am` is kinda a bit overkill (at least for now).

Yep - makes more sense once we have the PCP front-end of course.

Reply:
ACK. That's what I meant with the parens …